src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -98,6 +99,12 @@ public NumericFieldSet getExtraNumericFields() {
     public void setExtraNumericFields(NumericFieldSet extraNumericFields) {
         this.extraNumericFields = extraNumericFields;
     }
+
+    public Date getLastAudited() {
```

Comment:
I understand this is just a starter modification, but we should keep NIH-specific things out of the `StreetEdge` class.  Maybe for now could put this function on the `PlanGenerator`, which also will need to have NIH-specific factored out once the structure is in place.